File: Web.Admin/ClientApp/src/app/models/testSession/TestSessionLite.ts
import { TestSessionStatus } from './TestSessionStatus';

export class TestSessionLite {

  testSessionId: number;
  created: Date;
  status: TestSessionStatus;

  public TestSessionLite(testSessionId: number,
    created: Date,
    status: TestSessionStatus) {
    this.testSessionId = testSessionId;
    this.created = created;
    this.status = status;
  };

}

File: Libraries/Shared/NIST.CVP.Libraries.Shared.ACVPCore.Abstractions/Models/TestSessionLite.cs
using System;

namespace NIST.CVP.Libraries.Shared.ACVPCore.Abstractions.Models
{
    public class TestSessionLite
    {
        public long TestSessionId { get; set; }
        public DateTime Created { get; set; }
        public TestSessionStatus Status { get; set; }
    }
}